packages/core/src/external/athenahealth/index.ts

```diff
@@ -576,23 +584,23 @@ class AthenaHealthApi {
   async getAppointmentsFromSubscription({
     cxId,
     departmentIds,
-    startLastModifiedDate,
-    endLastModifiedDate,
+    showProcessedStartDateTime,
+    showProcessedEndDateTime,
```

Comment:
nit: this feels like a boolean/flag, not the start of a date range

I understand this is a parameter of the EHR, but changing this function's param names we're bringing it into our codebase.

Wouldn't `lastModifiedDateStart` and `lastModifiedDateEnd` be simpler to read/understand?

Reply:
Changed to be more usable.